aws-cli/README.rst

```diff
@@ -26,23 +27,25 @@ To run these examples, you'll need:
 Running the examples
 ====================
 
-Examples are typically written as functions in shell script files that can be sourced from other 
-files. Most are accompanied by a unit test script in the **tests** directory that you can run to validate
-that each example works. The test scripts create the include setup and teardown functions that create
-and destroy any prerequisite resources. We take care that all AWS resources that the example creates
-are also destroyed to avoid incurring any unwanted costs. We do recommend when you're done with an example
-that you check the resources in your account to ensure that the teardown function 
-worked as expected and didn't accidentally leave anything behind.
-
-To run the examples, you need to create a shared credentials file. For more information about how
-to set up a shared credentials file, see `Configuration and Credential File Settings <
+Examples are typically written as functions in shell script files that can be
+sourced from other files. Most are accompanied by a unit test script that you 
+can run to validate that each example works. The test scripts create the 
+include setup and teardown functions that create and destroy any AWS resources

```

Comment:
"The test scripts create the include setup and teardown" -> "The test scripts include setup and teardown"

Reply:
Fixed.